### DISCUSSION TITLE
iStore里面BaiduPCS-Web 已经不维护了，能替换成docker版的不？

### DISCUSSION THREAD

**Discussion by jejmelody** [istoreos/istoreos#907] [Ideas]
项目地址：
https://github.com/john-shine/Docker-CodeWeavers_CrossOver-VNC/tree/master/BaiduNetdisk

**Comment by jjm2473** [comment]
这种使用桌面系统的容器不实用，镜像非常大，而且一般的软路由不够性能来串流桌面。

其次就是这个docker镜像只有x86版本，没法支持arm64平台。

**Comment by w9315273** [comment]
docker版本就维护吗？3 years ago
